### PR TITLE
Update billing section in docs

### PR DESCRIPTION
**Pull request by aayushmahapatra**

### CODE REVIEW

File: pages/billing/upgrade/credit-card.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Follow these steps to upgrade to a Pro plan:
+
+1. Upgrade Plan:
+   - Click "Upgrade Plan" under the **Billing** tab.
+   - Click "Add Payment Method" and then, Click "Credit / Debit Card".
+2. Add Credit Card:
+   - Fill in the details and Click "Attach Payment Method".
+3. Plan Details:
+   - If you have a coupon code, you can add it in the **Coupon code** section and Click on the arrow button to activate it.
```
Comment (rekpero) [inline pages/billing/upgrade/credit-card.mdx:14]: remove the coupon code section from everywhere. we don't have coupon code on our plan upgrade page

Comment (rekpero) [reply pages/billing/upgrade/credit-card.mdx:14]: do it from other places as well

Comment (rekpero) [reply pages/billing/upgrade/credit-card.mdx:14]: Also create another PR for applying credit on org

**Review by rekpero — CHANGES_REQUESTED** [review]
check my comments

File: pages/billing/upgrade/credit-card.mdx (hunk 1 above)
Comment (aayushmahapatra) [reply pages/billing/upgrade/credit-card.mdx]: Didn't get what you mean by this?

Comment (rekpero) [reply pages/billing/upgrade/credit-card.mdx]: bro check the app now. when you are in the plan upgrade section now there is no coupon section to add any type of coupon. we have remove the coupon logic with a credit based system. didn't you know about this feature yet?

Comment (aayushmahapatra) [reply pages/billing/upgrade/credit-card.mdx]: Oh! You wrote credit so got confused cause on the platform it's Coupon Credit.

**Pull request merged**